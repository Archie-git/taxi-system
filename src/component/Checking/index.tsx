import React from 'react'
import { Container, Box, CircularProgress, Typography } from '@mui/material'

const Checking: () => JSX.Element = () => {
  return (
    <Container sx={style.container}>
      <Box sx={style.box}>
        <CircularProgress />
        <Typography variant="h4" sx={style.text}>Checking...</Typography>
      </Box>
    </Container>
  )
}

export default Checking

const style = {
  container: {
    height: '100vh',
    paddingTop: '30vh'
  },
  box: {
    display: 'flex',
    justifyContent: 'center'
  },
  text: {
    textIndent: '1rem'
  }
}
