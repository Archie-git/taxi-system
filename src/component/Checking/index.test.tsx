import React from 'react'
import renderer from 'react-test-renderer'
import Checking from './index'

it('Test Checking', () => {
  const tree = renderer
    .create(<Checking />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
