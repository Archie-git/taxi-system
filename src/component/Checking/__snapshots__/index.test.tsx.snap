// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Checking 1`] = `
<div
  className="MuiContainer-root MuiContainer-maxWidthLg css-1iior23-MuiContainer-root"
>
  <div
    className="MuiBox-root css-1l4w6pd"
  >
    <span
      className="MuiCircularProgress-root MuiCircularProgress-indeterminate MuiCircularProgress-colorPrimary css-18lrjg1-MuiCircularProgress-root"
      role="progressbar"
      style={
        {
          "height": 40,
          "width": 40,
        }
      }
    >
      <svg
        className="MuiCircularProgress-svg css-1idz92c-MuiCircularProgress-svg"
        viewBox="22 22 44 44"
      >
        <circle
          className="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate css-176wh8e-MuiCircularProgress-circle"
          cx={44}
          cy={44}
          fill="none"
          r={20.2}
          strokeWidth={3.6}
          style={{}}
        />
      </svg>
    </span>
    <h4
      className="MuiTypography-root MuiTypography-h4 css-1arwu0y-MuiTypography-root"
    >
      Checking...
    </h4>
  </div>
</div>
`;
